src/export-page/export-stepper/ExportStepper.jsx

```diff
@@ -33,7 +34,7 @@ const ExportStepper = ({ intl, courseId }) => {
     const id = setInterval(() => {
       if (isStopFetching) {
         clearInterval(id);
-      } else {
+      } else if (savingStatus === RequestStatus.SUCCESSFUL) {
```

Comment:
lets add here 
`dispatch(updateSavingStatus({ status: '' }));`

Reply:
Why? I think this is incorrect place for this operation, because component about Stepper shouldn't have logic about API status